bin/mocha-current-file.js

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env node
+
+// Copyright IBM Corp. 2018. All Rights Reserved.
+// Node module: @loopback-next
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+/*
+========
+This is used in the launch.json to enable you to debug a test file written in
+typescript.  This function attempts to convert the passed typescript file to
+the best-guess output javascript file.
+
+It walks up the filesystem from the current file, stops at package.json, and
+looks in `dist`
+
+Ideally, we could somehow use the typescript compiler and tsconfig.json to get
+the explicit output file instead of trying to guess it, but there might be
+overhead.
+
+Ex:
+```jsonc
+{
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "type": "node",
+      "request": "launch",
+      "name": "Debug Current Test File",
+      "program": "${workspaceRoot}/bin/mocha-current-file",
+      "cwd": "${workspaceRoot}",
```

Comment:
Should this block be identical to launch.json? runtimeArgs is missing.

Reply:
Yes please, let's include `runtimeArgs` too.

Reply:
Added!